[Processing] add missing parameters to random points in extent (#6004)
qgis/QGIS-Documentation: docs/user_manual/processing_algs/qgis/vectorcreation.rst

       Default: *Project CRS*
     - CRS of the random points layer
   * - **Maximum number of search attempts given the minimum distance**
     - ``MAX_ATTEMPTS``
     - [number]

       Default: 200

>      - Maximum number of attempts if minimum distance it too small

Reviewer: ```suggestion
     - Maximum number of attempts to place the points
```
You would likely need more attempts to place the points if the minimum distance is big than if it's too small. But I don't think this parameter depends on the set value of min distance; AFAIU the alg will run until either it places all the expected points or hits this max of attempts.

Author: Maybe I think we understood the same thing but explained in a different way :-)  Anyway, your description is more clear. Thanks!
